src/scala/ly/stealth/mesos/kafka/Executor.scala

```diff
@@ -92,6 +95,25 @@ object Executor extends org.apache.mesos.Executor {
       }
     }
 
+    def startCollectingMetrics(): Unit = {
+      new Thread {
+        setName("BrokerMetrics")
+        def send(driver: ExecutorDriver, metrics: Broker.Metrics): Unit = {
+          driver.sendFrameworkMessage(JSONObject(Map("metrics" -> metrics.toJson)).toString().getBytes)
+        }
+        override def run(): Unit = {
+          while (true) {
+            try {
+              BrokerServer.Metrics.collect.foreach(metrics => send(driver, metrics))
+              Thread.sleep(5000L)
+            } catch {
+              case e: Throwable => logger.warn("", e)
```

Comment:
@olegkovalenko Also please:
- replace Option[Broker.Metrics] with Broker.Metrics cause we are using nulls across the code and do not use scala Options by convention.